str.d.ts

```diff
@@ -0,0 +1,21 @@
+import type {Props} from './dom.d.ts';
+
+export {Raw, boolAttrs, voidElems, cls, len, /* TODO: map */} from './dom.d.ts';
+
+type Child = string | Child[];
```

Comment:
This should be something like: `Stringable | Child[]` where `Stringable = Prim | StringableObject`.